Src/ILGPU.Algorithms/Runtime/Cuda/API/NvmlAPI.xml

```diff
@@ -0,0 +1,609 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ImportLibrary Namespace="ILGPU.Runtime.Cuda.API"
+               ClassName="NvmlAPI"
+               VersionType="NvmlAPIVersion"
+               DefaultMethodAccessModifier="public"
+               DefaultReturnType="NvmlReturn">
+    <LibraryVersions>
+        <LibraryVersion Name="V6" Platform="Windows" LibName="nvml" />
```

Comment:
QUESTION: Is V6 the correct version number?
This [link](https://developer.nvidia.com/nvidia-management-library-nvml) only lists "Latest Production" and "Released with CUDA 5.5".

Reply:
Seems to work without any issues on an NVIDIA DGX machine.